app/views/users/show.html.erb

```diff
@@ -1 +1,21 @@
 <h1>User Profile</h1>
+<div>
+  <div> <%= image_tag @user.photo, alt: "picture of user", style: "width: 50px; height: 50px;" %> </div>
+  <div>
+    <h2> <%= @user.name %> </h2>
+    Posts Counter: <%= @user.posts_counter %>
+  </div>
+
+  <div> Bio: <br> <%= @user.bio %> </div>
+
+  <ul>
+    <% @user.recent_posts.each_with_index do |post, index| %>
+      <li>
+        Post #<%= index + 1 %> <br>
+        <%= link_to post.title, user_posts_path(@user) %>
```

Comment:
- [ ] Good job adding a path to posts, however, instead of putting the path on to see all posts on the post title, it could be better to add a `See all Post`  button to direct to All Posts, because as I have seen when you click on any post title, inside of directing to the `show post` page, it is directing to the `all posts` page. Kindly change the line `15`  `<%= link_to post.title, user_posts_path(@user) %>` to this: `<%= link_to post.title, user_post_path(@user) %>` then add the button to see all posts at the end of the page, so we will have something like this:

````ruby
<h1>User Profile</h1>
<div>
  <div> <%= image_tag @user.photo, alt: "picture of user", style: "width: 50px; height: 50px;" %> </div>
  <div>
    <h2> <%= @user.name %> </h2>
    Posts Counter: <%= @user.posts_counter %>
  </div>

  <div> Bio: <br> <%= @user.bio %> </div>

  <ul>
    <% @user.recent_posts.each_with_index do |post, index| %>
      <li>
        Post #<%= index + 1 %> <br>
        <%= link_to post.title, user_post_path(@user) %>
        <br>
        Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
      </li>
    <% end %>
  </ul>

  <button>
    <%= link_to 'See all Post', user_posts_path(@user) %>
  </button>

</div>
````

![blog_user_page](https://github.com/Omwomo/eugene-mindscape/assets/57408419/a2401890-ef74-480a-98c3-4ca3cee3e4cb)